src/main/java/ladder/domain/Direction.java

```diff
@@ -0,0 +1,45 @@
+package ladder.domain;
+
+import ladder.constants.Move;
+
+import java.util.Objects;
+
+public class Direction {
+  private final Move value;
```

Comment:
```suggestion
public class Direction {
  private final Move move;
```

기왕 Move 라는 enum 타입을 만들었으니, 변수명도 의미를 가질 수 있게끔 move 로 하는 것은 어떠신가요?
value 는 역할과 책임을 유추하기 어려운 매우 안 좋은 변수 네이밍입니다.

Reply:
@ghojeong

정완님 ㅎㅎ 저는 래퍼 클래스에 대한 멤버 변수명에 대한 고민이 많았는데요~

제가 생각하기엔 역할과 책임은 클래스에 있고
Direction.move 보단 Direction.value 이 더 명확하다고 생각하는데 어떻게 생각하시나요 ?

Reply:
흠... 의미론적으로 생각하면 그런 관점도 합당하군요.
넵. 이 부분은 굳이 수정하지 않으셔도 괜찮을것 같습니다.